Gemfile.lock

```diff
@@ -29,43 +29,23 @@ GEM
     danger-shroud (0.0.6)
       danger-plugin-api (~> 1.0)
       nokogiri
-    faraday (1.10.2)
-      faraday-em_http (~> 1.0)
-      faraday-em_synchrony (~> 1.0)
-      faraday-excon (~> 1.1)
-      faraday-httpclient (~> 1.0)
```

Comment:
So these all have the correct versions in the new version of `faraday` then?

Reply:
Should be, I did a `bundle update` to make these changes